README.rst

```diff
@@ -4,7 +4,8 @@ What is borg-import?
 borg-import can import backups from some other backup software, currently supported are:
 
 - rsnapshot
-- simple rsync+hardlinks (One folder per archive, using folder mtime)
+- ``rsynchl``: simple rsync+hardlinks (One folder per archive, using folder mtime)
```

Comment:
maybe rather use `rsync-hl` or `rsync+hardlinks` as shortcut for this.

having the char sequence `synch` (-> synchronization) in there makes it difficult to see what it means for people who do not know already.

Reply:
I added `rsynchl` because I wanted to directly give users the mode to call with borg-import. I agree that `rsynchl` should might get renamed to something like `rsync-hl`, but this is out of scope of this PR.